recipes/pyrad/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: pyrad
+  version: '3.0.64'
+
+source:
+  fn: 3.0.64.tar.gz
+  url: https://github.com/dereneaton/pyrad/archive/3.0.64.tar.gz
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - six
+    - muscle
+    - vsearch
+  run:
+    - python
+    - numpy
+    - scipy
+    - six
+    - muscle
+    - vsearch
+
+
+test:
+  imports:
+    - pyrad
```

Comment:
A few more imports would be nice to see if potential modules are installed as well. For example:

import pyrad.packet
import pyrad.client
import pyrad.dictionary